usmqe_tests/api/tendrl/test_volume.py

```diff
@@ -0,0 +1,280 @@
+def test_delete_volume_invalid(valid_cluster_id, invalid_volume_id):
+    """@pylatest api/gluster.delete_volume
+        API-gluster: delete_volume
+        ******************************
+
+        .. test_metadata:: author fbalak@redhat.com
+
+        Description
+        ===========
+
+        Delete gluster volume ``Vol_test`` via API.
+
+        .. test_step:: 1
+
+                Connect to Tendrl API via POST request to ``APIURL/:cluster_id/GlusterDeleteVolume``
+                Where cluster_id is set to predefined value.
+
+        .. test_result:: 1
+
+                Server should return response in JSON format:
+
+                Return code should be **202** with data ``{"message": "Accepted"}``.
+                """
+    api = tendrlapi.ApiGluster()
+    volume_data = {
+        "Volume.volname": valid_cluster_id,
+        "Volume.vol_id": invalid_volume_id
+    }
+
+    job_id = api.delete_volume(valid_cluster_id, volume_data)["job_id"]
+    # TODO check correctly server response or etcd job status
+    api.wait_for_job_status(
+            job_id,
+            status="failed",
+            issue="https://github.com/Tendrl/tendrl-api/issues/33")
+
+
+def test_delete_volume_valid(valid_cluster_id, valid_volume_name, valid_volume_id):
+    """@pylatest api/gluster.delete_volume
+        API-gluster: delete_volume
+        ******************************
+
+        .. test_metadata:: author fbalak@redhat.com
+
+        Description
+        ===========
+
+        Delete gluster volume ``Vol_test`` via API.
+
+        .. test_step:: 1
+
+                Connect to Tendrl API via POST request to ``APIURL/:cluster_id/GlusterDeleteVolume``
+                Where cluster_id is set to predefined value.
+
+        .. test_result:: 1
+
+                Server should return response in JSON format:
+
+                Return code should be **202** with data ``{"message": "Accepted"}``.
+                """
+    api = tendrlapi.ApiGluster()
+    volume_data = {
+        "Volume.volname": valid_volume_name,
+        "Volume.vol_id": valid_volume_id
+    }
+
+    job_id = api.delete_volume(valid_cluster_id, volume_data)["job_id"]
+    api.wait_for_job_status(job_id, issue="https://github.com/Tendrl/api/issues/33")
+    """@pylatest api/gluster.create_volume
+        API-gluster: create_volume
+        ******************************
+
+        .. test_metadata:: author fbalak@redhat.com
+
+        Description
+        ===========
+
+        Check if there is created volume on gluster nodes via CLI.
+
+        .. test_step:: 1
+
+            Connect to gluster node machine via ssh and run
+            ``gluster volume info command``
+
+        .. test_result:: 1
+
+            There should be listed gluster volume named ``Vol_test``.
+
+            """
+    test_gluster = gluster.GlusterCommon()
+    test_gluster.find_volume_name(valid_volume_name, False)
+    deleted = api.get_volume_list(valid_cluster_id)[0][valid_volume_id]["deleted"]
```

Comment:
please put here some comment as we don't know if there should be deleted parameter or if it should be deleted from etcd